### ISSUE TITLE
Rule proposal: Sort Enum And Interface Keys

### ISSUE THREAD

**Issue by PodaruDragos** [typescript-eslint/typescript-eslint#4008]
### Description

It would be great it a rule existed where interfaces and enum keys can be sorted.
If this is not considered a thing that might be added, feel free to close this issue

### Fail

```ts
  interface A {
    b: string;
    a: string;
  }
  
  // eg: Expected a to come before b
  
```

### Pass

```ts
interface A {
  a: string;
  b: string;
}
```

**Comment by JoshuaKGoldberg** [comment]
Aha, I've been using that plugin in http://github.com/JoshuaKGoldberg/template-typescript-node-package. It's great. cc @azat-io 😄 

Let's close this out, yes. Per https://typescript-eslint.io/linting/troubleshooting/formatting we're trying not to take on additional responsibility around stylistic concerns. And the plugin does a great job on its own. Thanks for the spot!
